tests/test_cryptocurrency/test_coinpaprika_view.py

```diff
@@ -2,102 +2,134 @@
         self.assertIn("market_cap_usd", capt)
 
     def test_coins(self):
         capturedOutput = io.StringIO()
         sys.stdout = capturedOutput
-        coinpaprika_view.coins([])
+        coins([])
         sys.stdout = sys.__stdout__
         capt = capturedOutput.getvalue()
         self.assertIn("rank", capt)
 
+    def test_all_coins_market_info(self):
+        capturedOutput = io.StringIO()
+        sys.stdout = capturedOutput
+        all_coins_market_info([])
+        sys.stdout = sys.__stdout__
+        capt = capturedOutput.getvalue()
+        self.assertIn("Displaying data vs USD", capt)
+
     def test_all_coins_info(self):
         capturedOutput = io.StringIO()
         sys.stdout = capturedOutput
-        coinpaprika_view.all_coins_info([])
+        all_coins_info([])
         sys.stdout = sys.__stdout__
         capt = capturedOutput.getvalue()
         self.assertIn("Displaying data vs USD", capt)
 
     def test_all_exchanges(self):
         capturedOutput = io.StringIO()
         sys.stdout = capturedOutput
-        coinpaprika_view.all_exchanges([])
+        all_exchanges([])
         sys.stdout = sys.__stdout__
         capt = capturedOutput.getvalue()
         self.assertIn("Displaying data vs USD", capt)
 
     def test_search(self):
         capturedOutput = io.StringIO()
         sys.stdout = capturedOutput
-        coinpaprika_view.search(["-q", "bt"])
+        search(["-q", "bt"])
         sys.stdout = sys.__stdout__
         capt = capturedOutput.getvalue()
         self.assertIn("category", capt)
 
     def test_all_platforms(self):
         capturedOutput = io.StringIO()
         sys.stdout = capturedOutput
-        coinpaprika_view.all_platforms([])
+        all_platforms([])
         sys.stdout = sys.__stdout__
         capt = capturedOutput.getvalue()
         self.assertIn("platform_id", capt)
 
     def test_contracts(self):
         capturedOutput = io.StringIO()
         sys.stdout = capturedOutput
-        coinpaprika_view.contracts([])
+        contracts([])
         sys.stdout = sys.__stdout__
         capt = capturedOutput.getvalue()
         self.assertIn("active", capt)
 
     def test_find(self):
         capturedOutput = io.StringIO()
         sys.stdout = capturedOutput
-        coinpaprika_view.find(["-c", "BTC"])
+        find(["-c", "BTC"])
         sys.stdout = sys.__stdout__
         capt = capturedOutput.getvalue()
         self.assertIn("index", capt)
 
     def test_twitter(self):
         capturedOutput = io.StringIO()
         sys.stdout = capturedOutput
-        coinpaprika_view.twitter("eth-ethereum", [])
+        twitter("eth-ethereum", [])
         sys.stdout = sys.__stdout__
         capt = capturedOutput.getvalue()
         self.assertIn("Couldn't find", capt)
 
     def test_events(self):
         capturedOutput = io.StringIO()
         sys.stdout = capturedOutput
-        coinpaprika_view.events("eth-ethereum", [])
+        events("eth-ethereum", [])
```

Comment:
I would leave here the mention to the modules, it makes it easier to understand where it is.

Also, I had to remove 3 crypto tests that were causing tests to fail stochastically. This is due to performing api requests on a global level, hence why it's best to have these commnand contained as we have discussed  before :)